sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClient.cs

```diff
@@ -234,7 +234,8 @@ private AnswerCallRequestInternal CreateAnswerCallRequest(AnswerCallOptions opti
                 request.AzureCognitiveServicesEndpointUrl = options.AzureCognitiveServicesEndpointUrl.AbsoluteUri;
             }
             request.MediaStreamingConfiguration = CreateMediaStreamingOptionsInternal(options.MediaStreamingOptions);
-            request.AnsweredByIdentifier = Source == null ? null : CommunicationIdentifierSerializer.Serialize(Source);
+
+            request.AnsweredByIdentifier = Source == null ? null : CommunicationIdentifierSerializer.Serialize(Source).CommunicationUser;
```

Comment:
Hi @huachuandeng,

Kindly review changes in this file which I made, there are resulted from[ your commit](https://github.com/williamzhao87/azure-rest-api-specs/commit/366f31cb260cb6c0ef4c8964440e9fee25bdf39f) in swagger

Reply:
My change is just for source in callconnectionproperties but my team does introduce some change here and my teammate is making change on dot net sdk

Reply:
We also made change on create call and callconnectionProperties if you don't mind could you wait for my team's change in?

Reply:
okey thanks for info!